src/test/kotlin/com/celuveat/member/adapter/in/rest/SocialLoginControllerTest.kt

```diff
@@ -55,6 +59,34 @@ class SocialLoginControllerTest(
             }
         }
     }
+
+    context("소셜 로그인 URL을 요청한다") {
+        val socialLoginType = SocialLoginType.KAKAO
+        val requestOrigin = "http://localhost:3000"
+        val socialLoginUrl = "https://social.com/authorize?redirect_uri=$requestOrigin&client_id=clientId"
+
+        test("소셜 로그인 URL을 성공적으로 반환한다") {
+            every { getSocialLoginUrlUseCase.getSocialLoginUrl(socialLoginType, requestOrigin) } returns socialLoginUrl
+
+            mockMvc.get("/social-login/login/{socialLoginType}/url", socialLoginType) {
```

Comment:
```suggestion
            mockMvc.get("/social-login/{socialLoginType}/url", socialLoginType) {
```
uri에 login이 두번 겹치는데 이것도 지우면 어떨까요?!

Reply:
오 좋아요!!!!